frontend/src/ProfilePage/FormPages/PageOne.css

```diff
@@ -0,0 +1,46 @@
+.form-page label {
```

Comment:
nit: see the camelCase comment from earlier. you could go with hyphens as well-- let's just be consistent